docs/essentials/viewports.md

```diff
@@ -0,0 +1,330 @@
+---
+title: 'Viewports'
+---
+
+<div style="background-color:#F8FAFC">
+TODO: CLEANUP comments once this is vetted.
+</div>
+
+The Viewports Essential toolbar item allows you to adjust the viewport your story is rendered in via Storybook’s UI:
+
+<div style="background-color:#F8FAFC">
+TODO: add Gif mentioned in the SB 6.0 needs to be further vetted
+</div>
+
+## Configuration
+
+By default, you are presented with a set of common viewports. 
+
+If you want to change the default set of viewports, you can set the global `parameters.viewport` [parameter](../writing-stories/parameters) in your [`.storybook/preview.js`](../configure/overview#configure-story-rendering):
+
+```js
+// .storybook/preview.js
+export const parameters: {
+  viewport: {
+    viewports: newViewports, // newViewports would be an ViewportMap. (see below for examples)
+    defaultViewport: 'someDefault',
+  },
+}
+```
+
+The viewport global can take a object with the following keys:
+
+| Field                  | Type          | Description                                            |Default Value |
+| -----------------------|:-------------:|:------------------------------------------------------:|:------------:|
+| **defaultViewport**    | String        |Sets the default viewport                               |`'responsive'`|
+| **disable**            | Boolean       |Disables the viewport                                   |N/A           |
+| **viewports**          | Object        |The configuration object for the viewport               |`{}`          |
+
+
+The viewports object needs the following keys:
+
+| Field                  | Type          | Description                                            |Example values                             |
+| -----------------------|:-------------:|:-------------------------------------------------------|:-----------------------------------------:|
+| **name**               | String        |Name for the viewport                                   |`'Responsive'`                             |
+| **styles**             | Object        |Sets Inline styles to be applied to the story           |`{width:0,height:0}`                       |
+| **type**               | String        |Type of the device (e.g. desktop, mobile, or tablet)    |`desktop`                                  |
+
+<!-- 
```

Comment:
What's up with this commented out section?

Reply:
That was left behind based on item on top of the page
```
<div style="background-color:#F8FAFC">
TODO: CLEANUP comments once this is vetted.
</div>
```
section was cleaned up